exchanges/gateio/gateio_wrapper.go

```diff
@@ -2398,8 +2405,17 @@ func (g *Gateio) deriveSpotWebsocketOrderResponses(responses []*WebsocketOrderRe
 		return nil, common.ErrNoResponse
 	}
 
-	out := make([]*order.SubmitResponse, 0, len(responses))
-	for _, resp := range responses {
+	out := make([]*order.SubmitResponse, len(responses))
+	for i, resp := range responses {
+		if resp.Label != "" { // Only present in a multi order response
```

Comment:
Does the response struct that has errored have no other information? Seeing that it has an error is good, but having no other information isn't very handy for seeing why it errored. Please add as many fields that are populated as possible

Reply:
All it returns: `{"text":"t-742","label":"BALANCE_NOT_ENOUGH","message":"Not enough balance"}`

But you can match it externally from your outbound `order.Submit` structs they come back in order. Or I pass in a slice of order submits into the param and then populate it from that if you want?

Reply:
If its still matchable that's fine